dspace-api/src/test/resources/test-config.properties

```diff
@@ -13,3 +13,4 @@ test.folder = ./target/testing/
 test.bitstream = ./target/testing/dspace/assetstore/ConstitutionofIreland.pdf
 test.exportcsv = ./target/testing/dspace/assetstore/test.csv
 test.importcsv = ./target/testing/dspace/assetstore/testImport.csv
+test.importpersoncsv = ./target/testing/dspace/assetstore/testImportPerson.csv
```

Comment:
@Raf-atmire : This test file does not appear to be used anywhere?  Am I misunderstanding how/where this configuration is used?  I'd expect it to appear in one of our IT classes.